dds_web/api/project.py

```diff
@@ -229,37 +229,29 @@ class GetPrivate(flask_restful.Resource):
     @logging_bind_request
     def get(self):
         """Get private key from database"""
-
-        project = project_schemas.ProjectRequiredSchema().load(flask.request.args)
-
-        # TODO (ina): Change handling of private key -- not secure
         flask.current_app.logger.debug("Getting the private key.")
```

Comment:
I think this was at a better place before, because if ProjectRequiredSchema is not satisfied, no further action is taken so not "Getting the private key."

Reply:
Yeah, fair enough! Changing